### ISSUE TITLE
Having problems with named params in annotations

### ISSUE THREAD

**Issue by Quaiks** [scalameta/paradise#35]
Hi!

I am trying to extract the value of a named parameter this way but I don't know if I am doing it right or this is not supported yet. Using scala.meta `1.1.0` and macro paradise `3.0.0-M5`

```scala
@namedParam(some = "text")
class SomeClass
```
```scala
final class namedParam(some: String) extends scala.annotation.StaticAnnotation {
  inline def apply(defn: Any): Any = meta {
    val q"new $_(some = $string)" = this
    defn
  }
}
```

The above code doesn't compile, throws the following error

```scala
Error:scalac: Error: unsupported tree AssignOrNamedArg(Ident(TermName("some")), Literal(Constant("text")))
(AssignOrNamedArg) some = "text"
(Apply) namedParam(some = "text")
(Template) Template(List(), List(namedParam(some = "text")), val _ = _, None)
(Apply) new namedParam(some = "text")
org.scalameta.paradise.converters.ConvertException: unsupported tree AssignOrNamedArg(Ident(TermName("some")), Literal(Constant("text")))
(AssignOrNamedArg) some = "text"
(Apply) namedParam(some = "text")
(Template) Template(List(), List(namedParam(some = "text")), val _ = _, None)
(Apply) new namedParam(some = "text")
```

Am I doing something wrong? Thanks in advance!

**Comment by Dveim** [comment]
Hi Quaiks!

Looks like this is one of missing parts in scala_reflect - scala_meta trees converter. I'll add this functionality soon.

**Comment by Quaiks** [comment]
That was quick! Thanks @Dveim. I am looking forward to it!

**Comment by Dveim** [comment]
See https://github.com/scalameta/paradise/pull/36 .

**Comment by Quaiks** [comment]
Nice!

**Comment by xeno-by** [comment]
Should be fixed in 3.0.0-SNAPSHOT.
